fastdtw.pyx

```diff
@@ -0,0 +1,135 @@
+#!python
```

Comment:
This didn't work for me when I tested it (it never imported the compiled version even when available). I ended up having to create the traditional "fastdtw/" folder and putting an __init__.py file in there with the above logic. This change is separated out into the 52a6dfb commit.